pkg/packagekit/autoupdates.jsx

```diff
@@ -241,7 +241,7 @@ const AutoUpdatesDialog = ({ backend }) => {
     function save(event) {
         setPending(true);
         backend.setConfig(enabled, type, day, time)
-                .always(Dialogs.close);
+                .finally(Dialogs.close);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19120-20230721-193803-d973134a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)